Makefile

```diff
@@ -23,18 +25,12 @@
 check: shellcheck shfmtcheck
 
 package:
-	sed "s/@@VERSION@@/$$(date '+%Y.%m.%d.%H')/" \
-		debian/changelog.in > debian/changelog
-
-	dpkg-buildpackage
-
-	@for ext in dsc tar.xz; do \
-		mv -v ../delphix-platform_*.$$ext artifacts; \
-	done
-
-	@for ext in buildinfo changes deb; do \
-		mv -v ../delphix-platform_*_amd64.$$ext artifacts; \
-	done
+	rm -rf artifacts
```

Comment:
why do we need to remove the `artifacts` directory and recreated it?

I think I liked it better when the directory existed in the repository, and `make package` didn't try to remove existing build products (i.e. how it currently works). We could add a `make clean` target if we need to clear that out (and `git clean -dxf` is insufficient).

Is there a specific reason for changing this?

Reply:
I felt like it simplified things since we don't need an extra `.gitignore` file in there, but it's not something I had a strong opinion so I reverted it back to what it was.